src/main/java/com/networknt/schema/utils/JsonSchemaRefs.java

```diff
@@ -33,7 +33,7 @@ public class JsonSchemaRefs {
      * @param schema the schema
      * @return the ref
      */
-    public static JsonSchemaRef from(JsonSchema schema) {
+    public static JsonSchemaRef extractSchemaRefFromSchema(JsonSchema schema) {

```

Comment:
I actually don't really see the need to rename `from` to `extractSchemaRefFromSchema`. The naming convention of using `from` or `of` is quite common. eg. https://docs.oracle.com/javase/8/docs/api/java/time/Period.html#from-java.time.temporal.TemporalAmount-.